doc/cephadm/mon.rst

```diff
@@ -4,8 +4,8 @@ MON Service
 
 .. _deploy_additional_monitors:
 
-Deploy additional monitors
-==========================
+Deploying additional monitors (beyond the default three)
```

Comment:
```suggestion
Deploying additional monitors
```

If a user adds `skip-mon-network`, he heeds to follow this guide in order to deploy more than the initial mon.